.travis.yml

```diff
@@ -0,0 +1,49 @@
+language: cpp
+
+dist: trusty
```

Comment:
Is there a newer dist on travis available? trusty is pretty old, xenial would be great